src/tests/hygiene/pymodule.rs

```diff
@@ -16,8 +16,20 @@ fn foo(_py: crate::Python<'_>, _m: &crate::types::PyModule) -> crate::PyResult<(
 #[crate::pymodule]
 #[pyo3(crate = "crate")]
 fn my_module(_py: crate::Python<'_>, m: &crate::types::PyModule) -> crate::PyResult<()> {
+    #[allow(deprecated)]
     m.add_function(crate::wrap_pyfunction!(do_something, m)?)?;
     m.add_wrapped(crate::wrap_pymodule!(foo))?;
 
     ::std::result::Result::Ok(())
 }
+
+#[crate::pymodule]
+#[pyo3(crate = "crate")]
+fn my_module_bound(_py: crate::Python<'_>, m: &crate::Bound<'_, crate::types::PyModule>) -> crate::PyResult<()> {
+    use crate::types::PyModuleMethods;
```

Comment:
Yeah, I can see your point here. These traits are all new, so I guess we need to figure out how we want to handle them. 

I guess the main point to test here is that the macro expansion compiles without additional imports. So fully qualified syntax seems appropriate here. 

Good catch with `PyAnyMethods` in `test_no_imports.rs` seems like I added that 😇, shame on me 😆. I think that's only needed for `test_basic`, so maybe we should move that local instead of moving `PyModuleMethods` global? That way we can maybe at least somewhat verify that macro expansion does not rely on those imports, while still keeping this close to what a user might write.

After writing this, I think I agree that we should not generate that import silently.

Reply:
I think most code won't actually need to explicitly add `PyAnyMethods` because it's in the prelude.